qiskit/transpiler/preset_passmanagers/default.py

```diff
@@ -98,6 +100,9 @@ def default_pass_manager_simulator(basis_gates):
         PassManager: A passmanager that just unrolls, without any optimization.
     """
     pass_manager = PassManager()
+
+    pass_manager.append(Decompose(InitializeGate))
+    pass_manager.append(RemoveResetInZeroState())
```

Comment:
This has to be in the Default pass manager too, not just for the simulators.

Reply:
Fixed in 52e02902